modules/silicon_design/main.tf

```diff
@@ -157,6 +168,13 @@ resource "google_project_iam_member" "sa_p_notebook_permissions" {
   role     = each.value
 }
 
+resource "google_project_iam_member" "sa_p_cloudbuild_permissions" {
```

Comment:
Deploying this resource is erroring out with below error:
```
│ Error: Request `Create IAM Members roles/compute.storageAdmin serviceAccount:812933074202@cloudbuild.gserviceaccount.com for project "radlab-silicon-design-vzu8"` returned error: Batch request and retried single request "Create IAM Members roles/compute.storageAdmin serviceAccount:812933074202@cloudbuild.gserviceaccount.com for project \"radlab-silicon-design-vzu8\"" both failed. Final error: Error applying IAM policy for project "radlab-silicon-design-vzu8": Error setting IAM policy for project "radlab-silicon-design-vzu8": googleapi: Error 400: Service account 812933074202@cloudbuild.gserviceaccount.com does not exist., badRequest
│ 
│   with google_project_iam_member.sa_p_cloudbuild_permissions["roles/compute.storageAdmin"],
│   on main.tf line 171, in resource "google_project_iam_member" "sa_p_cloudbuild_permissions":
│  171: resource "google_project_iam_member" "sa_p_cloudbuild_permissions" {
│ 
```

You will also need to create the Service Account 812933074202@cloudbuild.gserviceaccount.com via [google_service_account](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/google_service_account) resource before you add permissions to the same. You can see example on _Line 158_ of your main.tf config.

Reply:
This usually done by GCP as a side effect of enabling the Cloud Build API, should we "wait" until it shows up in the state?

Reply:
I agree. Considering `resource "google_project_iam_member" "sa_p_cloudbuild_permissions"` is depended on a resource (Cloud Build Service Account) which is not built via terraform config it would be a good idea to wait until the Service Account exists.

Reply:
fixed with https://github.com/GoogleCloudPlatform/rad-lab/pull/50/commits/c6cd9374addbe3c2b27a7cef5e26886321c8872e